homeassistant/components/modbus/strings.json

```diff
@@ -70,6 +70,10 @@
     }
   },
   "issues": {
+    "removed_lazy_error_count": {
+      "title": "`{config_key}` configuration key is being removed",
+      "description": "Please remove the `{config_key}` key from the {integration} entry in your configuration.yaml file and restart Home Assistant to fix this issue. All errors will be reported, as lazy_error_count is accepted but ignored"
+    },
```

Comment:
```suggestion
      "description": "Please remove the `{config_key}` key from the {integration} entry in your `configuration.yaml` file and restart Home Assistant to fix this issue. All errors will be reported, as `lazy_error_count` is accepted but ignored."
    },
```

Reply:
Done

Reply:
May be push the update?

Reply:
????

https://github.com/home-assistant/core/pull/105037/commits/5825cfd2351e8c41fe53e68551bab04ea0a8eb68

Reply:
Stupid github logic